src/mpcCoreKit.ts

```diff
@@ -335,7 +341,7 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
         if (this.isRedirectMode) return;
 
         this.updateState({
-          oAuthKey: this._getOAuthKey(loginResponse),
+          oAuthKey: this._getPostBoxKey(loginResponse),
```

Comment:
Should we rename `state.oAuthKey` to `state.postBoxKey`? It would more clear, I think. But does this cause any compatibility issues?

Reply:
yeah, thought same thing. although it's a bit confusing because now postbox key might be different than oAuthKey.
we could also check if state has oAuthKey when we load, and then set postBoxKey (so convert state on load).
but, yeah, we can also leave as it is. what do you think?

Reply:
@himanshuchawla009 any preferences?

Reply:
i think rather thn changing we can add a new variable in state for postboxKey and let oauthKey resolves with a new function called : `_getOAuthKey` since `_getOAuthKey` will return same key as postboxkey for  secp it will remain backward compatible for react native users and it also gives us opportunity to fix the variable naming internally in the sdk , as in currently metadata key is referring to oauthKey but it should resolve to postboxKey.

Reply:
Hm, didn't fully get your comment.

Note that `_getPostBoxKey` is a private function, so we don't need to worry about the name here. As it returns the postBoxKey now (instead of the `oAuthKey`), I renamed it.

Let me see if I understood your suggestions correctly:
You are suggesting we also add `state.postBoxKey` (in addition to `state.oAuthKey`). And then we also have a function `_getOAuthKey`? What I don't get here is that why would we need that? Internally, we just need to access `postBoxKey` here, so we don't care about `oAuthKey` at all.

Reply:
yes we dont care about oauthKey at all but just suggesting to keep it for backward compat as CW pointed out the react native might face issues if we remove it totally. Instead of removing we can mark it as deprecated and remove it later.